guides/micronaut-openapi-generator-server/java/docs/controllers/BooksController.md

```diff
@@ -0,0 +1,52 @@
+# BooksController
```

Comment:
Should we have this file as part of the complete solution? Or should we delete it?

Reply:
This could not be generated at all by setting `-D modelDocs=false -D apiDocs=false`, but by default is generated.
I would keep so that the directory structure would be the same as user would get if they generate it.

* The same is true for all `/docs` files.*